### PR TITLE
Vertex AI Model Registry - BQML and custom model management notebook 

### PR DESCRIPTION
**Pull request by inardini**
<br>
This PR will add a notebook to show the model versioning capabilities of Vertex AI Model Registry with BQML and custom models. The notebook uses this [public dataset](http://mlg.ucd.ie/datasets/bbc.html) of BBC news articles which has been already utilised in [this](https://www.cloudskillsboost.google/focuses/1749?parent=catalog) Google Cloud Self-packed lab. 
<br><br><br>

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb [hunk 1]
```diff
@@ -0,0 +1,2824 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>Split into 4 cells</p><p>Add BQ to Costs</p><p>Sklearn -&gt; scikit-learn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>combine the 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>combine 2 text cells. Remove 'Skip this step.'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>you will download -&gt; you download</p><p> </p><p>extrect -&gt; extract</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>Before to build -&gt; Before building</p><p> </p><p>Bucket -&gt; bucket</p><p> </p><p>Bigquery -&gt; BigQuery</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>define BOW</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>Artefact -&gt; Artifact</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>what is the purpose of this code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>Sklearn -&gt; scikit-learn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>avoid trailing commas:</p><p>equirements=NAIVE_TRAIN_REQUIREMENTS,</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='98'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>Sklearn -&gt; scikit-learn</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='106'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>Artefact -&gt; Artifact</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='107'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>combine into single code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='140'/>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>create a empty file</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_bqml_custom_model_versioning.ipynb:1]: <p>done</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**